ecs-init/cache/cache.go

```diff
@@ -91,6 +104,18 @@ func (d *Downloader) getRegion() string {
 	return d.region
 }
 
+// getBucketRegion returns a region that contains the agent's bucket
+func (d *Downloader) getBucketRegion() string {
+	region := d.getRegion()
+	if config.IsSupportedRegion(region) {
+		return region
+	}
+
+	log.Debugf("Current region does not contain the agent's s3 bucket, using the default region(%s) for downloader.", 
```

Comment:
nit, logging wording: `"Current region (%s) does not contain...", region, config.DefaultRegionName`